### PR TITLE
added values novelObjectRecognition and activePlaceAvoidance

### PR DESCRIPTION
**Pull request by ychae**

### CODE REVIEW

**Comment by amapeters** [comment]
I think we should discuss these on the annotation meeting next Tuesday. We need a source for rodent behavioral data, including active place avoidance test and novel object recognition - may be more coming up. Need a reference source

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -180,6 +180,16 @@
         "value": "questionnaire",
         "description": "A document with a set of printed or written questions with a choice of answers, devised for the purposes of a survey or statistical study.",
         "source": "http://purl.obolibrary.org/obo/OBI_0001000"
+      },
+      {
+        "value": "novelObjectRecognition",
```
Comment (kdaily) [inline synapseAnnotations/data/experimentalData.json]: novel object recognition

File: synapseAnnotations/data/experimentalData.json [hunk 2]
```diff
@@ -180,6 +180,16 @@
...
+        "description": "A commonly used behavioral assay for the investigation of various aspects of learning and memory in mice.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/28892027"
+      },
+      {
+        "value": "activePlaceAvoidanceTest",
```
Comment (kdaily) [inline synapseAnnotations/data/experimentalData.json]: actilve place avoidance test

Comment (kdaily) [reply synapseAnnotations/data/experimentalData.json]: or task? the description calls it 'task'.

**Comment by ychae** [comment]
We decided on the more general one so if other recognition tasks come through, this will encompass objects and others.

**Pull request merged**